bin/docker_build

```diff
@@ -28,8 +33,11 @@ Dir.chdir APP_ROOT do
   # ** Eventually we will keep the `base` and `build` images in a separate repo,
   #   build them in a separate pipeline, and just pull them down for development
   #   Note: building the production image here is not necessary for development work
-  { base: 'rails_base', build: 'rails_build', production: 'idp_app' }.each do |name, description|
+  { base: 'base', build: 'build', development: 'dev', production: 'idp' }.each do |name, description|
     puts "== Building the #{name} Docker image =="
-    run "docker build . --file #{name}.Dockerfile --tag identity-#{description} --no-cache"
+    run "docker build . --file #{name}.Dockerfile --tag logindotgov/#{description}:#{head}"
+
+    puts "== Pushing the #{name} image to Docker Hub =="
+    run "docker push logindotgov/#{description}:#{head}"
```

Comment:
Hmm, I get
```
denied: requested access to the resource is denied
command failed (pid 43797 exit 1): docker push logindotgov/base:cbe56fee4
```
When I get to the push command.

I thought maybe I needed to `docker login`, but that doesn't help.  Maybe permissions in DockerHub?

I had to `docker login docker.io`

Reply:
@ctro you should be good to go now.